website/website/assets/js/map-en.js

```diff
@@ -239,50 +240,9 @@ function buildMap(...charts){
 		document.querySelector("#loader1").classList.remove("spinner-1");
 		document.querySelector("#loader2").classList.remove("spinner-1");
 
-		// show selected region on a map
-		addMiniMap();
-		// 1=green, 2=yellow, 3=red
-		let traffic_lights;
-		switch (report["result"]["label"]) {
-		    case 'green':
-		        traffic_lights = GOOD_QUALITY;
-		        break;
-		    case 'yellow':
-		        traffic_lights = MEDIUM_QUALITY;
-		        break;
-		    case 'red':
-		        traffic_lights = BAD_QUALITY;
-		        break;
-		    default:
-		        traffic_lights = UNDEFINED_QUALITY;
-		        break;
+		if (report["result"]["html"]){
+			document.getElementById('resultSection').insertAdjacentHTML('afterbegin', report["result"]["html"]);
```

Comment:
I miss a `else` here. What should happen if the result doesn't contain a HTML result?